backend/compact-connect/stacks/persistent_stack/user_email_notifications.py

```diff
@@ -0,0 +1,73 @@
+from aws_cdk.aws_iam import ServicePrincipal
+from aws_cdk.aws_ses import EmailIdentity, Identity, ConfigurationSet, EmailSendingEvent, EventDestination
+from aws_cdk.aws_sns import Topic, Subscription, SubscriptionProtocol
+from aws_cdk.aws_route53 import TxtRecord, IHostedZone
+from aws_cdk.aws_kms import IKey
+
+from constructs import Construct
+
+
+class UserEmailNotifications(Construct):
+    """
+    This Construct leverages SES to set up an email notification system to send cognito user events from our custom
+    domain with necessary SPF, DKIM, and DMARC verification records.
+
+    The topic is set up to forward all bounce and complaint events to the provided email address.
+    """
+    def __init__(
+            self, scope: Construct, construct_id: str, *,
+            hosted_zone: IHostedZone,
+            environment_context: dict,
+            master_key: IKey,
+            **kwargs
+    ):
+        super().__init__(scope, construct_id, **kwargs)
+
+        domain_name = hosted_zone.zone_name
+        operation_email = environment_context['notifications']['ses_operations_support_email']
+
+
+        self.email_feedback_topic = Topic(self, "FeedbackTopic",
```

Comment:
Is this the formatting your linter uses? Not something I used to, but if that's what it enforces, all good.

Reply:
Yes, if the function call causes the line length to be too long, my formatter puts the kwargs on separate lines. If there is a different formatting style that is preferred, let me know and I can adjust my IDE settings for the project.

Reply:
Hmm strange. Is this consistent with the rules used by @jusdino? I guess I'm agnostic but would like us to be consistent. (Eventually, I hope we can commit our lint rules to the repo.)

Reply:
Our style rules are already in the repo [here](https://github.com/csg-org/CompactConnect/blob/development/backend/pylintrc) and this code does pass. For what it's worth, the [PEP8 guidelines](https://peps.python.org/pep-0008/#indentation) allow for a few different approaches to indentation (including this one), any of which may be best suited to a particular circumstance, so I don't know how more prescriptive we want to be. With these construct declarations, with function signatures of exactly two positional arguments followed by a list of keyword arguments, I've been fond of the aesthetic of a newline after the opening bracket, a line with the two positional arguments, then a new line for each keyword argument. Honestly, though, while I like the look of it, it is strictly the farthest departure from the PEP8 examples:
```python
topic = Topic(
    self, 'FeedbackTopic',
    display_name="Email Feedback Forwarding Topic",
    master_key=master_key,
    enforce_ssl=True
)
```

Reply:
Ah honestly that example is what I thought the "standard" was :')

Given our discussion at today's retro let's table this and we can address globally when we get a linter set up.